it_zwierzak/adoption_announcement/views.py

```diff
@@ -25,21 +27,25 @@ def adoption_form(request):
     return render(request, 'adoption_announcement/adoption_application.html')
 
 
-class AdoptionApplicationView(LoginRequiredMixin, FormView): #tylko dla zalogowanych
+def confirmation_adoption_application(request):
+    return render(request, 'confirmation_adoption_application.html')
+
+
+class AdoptionApplicationView(LoginRequiredMixin, FormView):
     template_name = 'adoption_announcement/adoption_application.html'
-    form_class = AdoptionForm
-    success_url = '/animals_to_adoption'
 
-def adoption_application_view(request):
-    form = AdoptionForm(request.POST or None)
-    if form.is_valid():
-        form.save()
+    def get(self, request):
+        form = AdoptionForm()
+        return render(request, self.template_name, {'form': form})
+
+    def post(self, request):
+        form = AdoptionForm(request.POST)
+        if form.is_valid():
+            text = form.cleaned_data
+            form = AdoptionForm()
+            return redirect('confirmation_adoption_application:confirmation_adoption_application')
+            args = {'form': form, 'text': text}
+            return render(request, self.template_name, args)
```

Comment:
coś tu się popierdzielło, te returny są na tym samym wcięciu. Poza tym zmienna text to nie text tylko dane z formularza, co tu się miało zadziać?